src/ontology/genepio-edit.owl

```diff
@@ -14413,6 +14442,15 @@ AnnotationAssertion(dc11:date obo:GENEPIO_0002229 "2019-04-17T22:47:51Z"^^xsd:da
 AnnotationAssertion(rdfs:label obo:GENEPIO_0002229 "PulseNet:UploadDate"@en)
 ClassAssertion(obo:GENEPIO_0002185 obo:GENEPIO_0002229)
 
+# Individual: obo:GENEPIO_0004434 (University of Manitoba (UM))
+
+AnnotationAssertion(obo:IAO_0000115 obo:GENEPIO_0004434 "A canadian public research university in the province of Manitoba."@en)
```

Comment:
Small thing: Canadian needs a a capital C!

Reply:
Just corrected that and pushed the changes.